docs/src/styles/docs/base.scss

```diff
@@ -1,10 +1,8 @@
-[data-amplify-theme] {
-  font-family: var(--amplify-fonts-default-static);
-}
-@supports (font-variation-settings: normal) {
-  [data-amplify-theme] {
-    font-family: var(--amplify-fonts-default-variable);
-  }
+@import url('https://rsms.me/inter/inter.css');
```

Comment:
What do you think of hosting this file in our repo so we don't take an external load-time dependency? My concern is if this file moves or downloading it is slow, it might block first render?

Reply:
Yea that is probably a better idea. Let me do that real quick

Reply:
@reesscot added the font into the docs site and tested it locally